docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -71,27 +68,27 @@ remain lazy::
     >>> cube.has_lazy_data()
     True
 
-This is referred to as **lazy evaluation**.
+The process by which the operation is deferred until the result is requested is
+referred to as **lazy evaluation**.
 
 Certain operations, including regridding and plotting, can only be run on real data.
 Calling such operations on lazy data will automatically realise your lazy data.
 
 You can also realise (and so load into memory) your cube's lazy if you 'touch' the data.
-This means directly accessing the data by calling ``cube.data``, as in the previous example.
+To 'touch' the data means directly accessing the data by calling ``cube.data``,
+as in the previous example.
 
 Core data
 ^^^^^^^^^
 
 Cubes have the concept of "core data". This returns the cube's data in its
-current state. If a cube has lazy data, calling the cube's
-:meth:`~iris.cube.Cube.core_data` method will return the cube's lazy dask array.
-If the cube has real data, calling the cube's
-:meth:`~iris.cube.Cube.core_data` method will return the cube's real NumPy array.
+current state:
 
-Calling the cube's :meth:`~iris.cube.Cube.core_data` method will not change the
-state of the cube's data.  Thus, if the cube's data is lazy then calling
-:meth:`~iris.cube.Cube.core_data` will return the cube's lazy data and not
-realise it.
+ * If a cube has lazy data, calling the cube's :meth:`~iris.cube.Cube.core_data` method
+   will return the cube's lazy dask array. Calling the cube's
+   :meth:`~iris.cube.Cube.core_data` method **will not realise** the cube's data.
```

Comment:
Using the words "it" or "this" when writing technical documentation can be dangerous. Implicitly they refer back to the last named object/concept in the prose. This is fine until you use "it" or "this" to refer back to an _earlier_ object/concept in the prose. For example:

"If your baby is off its milk, try boiling it."

would get you, and quite possibly the baby, in a lot of hot water... (Here "it" is obviously referring to the milk, not the baby, but it's amazing how the brain interprets that sentence.)

As such, I'll always repeat a concept even if the prose starts to get a bit heavy because the thing a reader of technical documentation values most is clarity over brevity.